lib/bolt/outputter.rb

```diff
@@ -14,6 +14,18 @@ def self.for_format(format)
     def initialize(stream = $stdout)
       @stream = stream
     end
+
+    def unwrap_execution_result(result)
+      ok_hash = result.ok_nodes.iterator.map do |node, output|
+        {node: node, status: 'finished', result: output}
+      end
+      err_hash = result.error_nodes.iterator.map do |node, output|
```

Comment:
Does this map work properly? I thought the result was getting restructured sort of like: 

```
{'key': 'val', '_error': {'msg': ....} } 
to
{'msg': .... , 'value': { 'key': 'val' } } 
```

Reply:
This approach totally breaks if there are some errored nodes, but it's not even getting this far. ok_nodes seems to error.

Reply:
The way I'm trying to use this seems to be missing some setup. ok_nodes works fine within a plan, but not called here.

Reply:
Won't currently unwrap an error, which is a separate `Error` type.